Makefile

```
@@ -0,0 +1,24 @@
+SHELL=/bin/bash -euo pipefail
+
+export GO111MODULE        := on
+export PATH               := .bin:${PATH}
+
+.PHONY: install
+install: package-lock.json  django-ory-cloud/poetry.lock 
+		npm i
+		cd django-ory-cloud && poetry install
+
+# .PHONY: build-examples
+# build-examples:
+# 		cd code-examples/protect-page-login/nextjs && npm run build
+# 		cd code-examples/protect-page-login/flutter_web_redirect && flutter build web --web-renderer html
+# 		cd code-examples/protect-page-login/vue && VUE_APP_API_URL=http://localhost:4007 VUE_APP_ORY_URL=http://localhost:3006 npm run build
+# 		cd code-examples/protect-page-login/react && npm run build
+
+.PHONY: test
+test: install .bin/ory
+		./src/scripts/test.sh
+
+.bin/ory: Makefile
+		bash <(curl https://raw.githubusercontent.com/ory/meta/master/install.sh) -d -b .bin ory v0.1.33
+		touch -a -m .bin/ory
```

`touch` can either change the access time (with `-a`) or the modification time (with `-m`). If you provide both switches, that's similar to providing no switches and letting `touch` change both times. Run `man touch` for more info on these switches.

```suggestion
		touch .bin/ory
```

---

Hm it says
`-m      Change the modification time of the file.  The access time of the file is not changed unless the -a flag is also specified.`
`-a      Change the access time of the file.  The modification time of the file is not changed unless the -m flag is also specified.`
I take from that if I want to change both the access and modification time I have to provide both switches?

---

The [man-page for touch](https://man7.org/linux/man-pages/man1/touch.1.html) says this:

> Update the access and modification times of each FILE to the current time.
> -a     change only the access time
> -m     change only the modification time

So, the behavior of touch is:
- provide no switches --> changes access and modification time
- provide `-a` --> change only access time
- provide `-m` --> change only modification time
- provide `-a` and `-m` --> change access and modification time

You can verify this behavior through these steps:

```sh
# change modification time
touch -m -t 201512180130.09 README.md
# change access time
touch -a -t 201512180130.09 README.md
# verify that both access and modification time for this file are now at 2015-12-18
stat README.md
# change both times
touch README.md
# verify that both access and modification times are now set to current
stat README.md
```